### PR TITLE
:memo:  Public API Documentation Updates

### PR DESCRIPTION
**Pull request by fahimmahmoodmir**
- Added the limitation of Public API for commands retentions
- Added the size limitation for Structured Content
- Added the Reblaze limitation for request failure
- Fixed IBM watson v2 formatting problem

### CODE REVIEW

**Comment by fahimmahmoodmir** [comment]
Rendering of the Page

Rendered_Doc

File: pages/Documents/ConversationalAI/Third-PartyBots/public-api.md [hunk 1]
```diff
@@ -878,5 +878,13 @@ Example response of a command returned by the API will look like this with state
...
 | 400  | Bad request - Problem with body or query parameters.                 |
 | 401  | Unauthorized - Invalid bearer token.                                 |
+| 403  | Forbidden - If the request was marked as security risk by Reblaze.   |
 | 404  | Not Found - If the provided conversation Id is invalid or not found. |
 | 500  | Internal server error.                                               |
+
+### Limitations
+
+<ul>
+    <li>To ensure the safety of the API we are using <a href="https://www.reblaze.com/">Reblaze</a>. if the request throws Security Violation Error (403 - Forbidden) reason could be that your request body contains information that caused Reblaze to mark your request positive as a security risk. To understand security concepts of Reblaze please refer to the official documentation <a href="https://gb.docs.reblaze.com/product-walkthrough/security/concepts">here</a>. You can also refer to the information on best practices to avoid the false positives marking of valid request <a href="https://gb.docs.reblaze.com/using-the-product/best-practices/dealing-with-false-positive">here</a></li>
```
Comment (akirilyuk) [inline pages/Documents/ConversationalAI/Third-PartyBots/public-api.md:888]: why are you using html instead of md?

Comment (fahimmahmoodmir) [reply pages/Documents/ConversationalAI/Third-PartyBots/public-api.md]: changed

**Review by akirilyuk — APPROVED** [review]

File: pages/Documents/ConversationalAI/Third-PartyBots/public-api.md [hunk 2]
```diff
@@ -878,5 +878,11 @@ Example response of a command returned by the API will look like this with state
...
+| 403  | Forbidden - If the request was marked as security risk by Reblaze.   |
 | 404  | Not Found - If the provided conversation Id is invalid or not found. |
 | 500  | Internal server error.                                               |
+
+### Limitations
+
+- To ensure the safety of the API we are using [Reblaze](https://www.reblaze.com/). if the request throws Security Violation Error (403 - Forbidden) reason could be that your request body contains information that caused Reblaze to mark your request positive as a security risk. To understand security concepts of Reblaze please refer to the official documentation [here](https://gb.docs.reblaze.com/product-walkthrough/security/concepts). You can also refer to the information on best practices to avoid the false positives marking of valid request [here](https://gb.docs.reblaze.com/using-the-product/best-practices/dealing-with-false-positive)
+- Our Public API only retains a maximum of last 200 commands **per bot/agent**. There could be the possibility that if a conversation is ongoing but you are not able to fetch the commands because of this max commands retention policy
```
Comment (GFdiablo117) [inline pages/Documents/ConversationalAI/Third-PartyBots/public-api.md:888]: Thus, there could be a possibility that an ongoing conversation will not be able to fetch the commands due to this max. commands retention policy.

File: pages/Documents/ConversationalAI/Third-PartyBots/public-api.md [hunk 3]
```diff
@@ -878,5 +878,11 @@ Example response of a command returned by the API will look like this with state
 | 200  | OK - request for the given API succeeded.                            |
 | 400  | Bad request - Problem with body or query parameters.                 |
 | 401  | Unauthorized - Invalid bearer token.                                 |
+| 403  | Forbidden - If the request was marked as security risk by Reblaze.   |
 | 404  | Not Found - If the provided conversation Id is invalid or not found. |
 | 500  | Internal server error.                                               |
+
+### Limitations
+
+- To ensure the safety of the API we are using [Reblaze](https://www.reblaze.com/). if the request throws Security Violation Error (403 - Forbidden) reason could be that your request body contains information that caused Reblaze to mark your request positive as a security risk. To understand security concepts of Reblaze please refer to the official documentation [here](https://gb.docs.reblaze.com/product-walkthrough/security/concepts). You can also refer to the information on best practices to avoid the false positives marking of valid request [here](https://gb.docs.reblaze.com/using-the-product/best-practices/dealing-with-false-positive)
```
Comment (GFdiablo117) [inline pages/Documents/ConversationalAI/Third-PartyBots/public-api.md:887]: **If** the request throws Security Violation Error (403 - Forbidden) **, the** reason

**Review by GFdiablo117 — CHANGES_REQUESTED** [review]

**Review by GFdiablo117 — APPROVED** [review]

**Pull request merged**